src/ol/PluggableMap.js

```diff
@@ -1326,6 +1325,9 @@ class PluggableMap extends BaseObject {
    */
   setTarget(target) {
     this.set(MapProperty.TARGET, target);
+    if (!target) {
```

Comment:
This could go to the `handleTargetChanged_` function instead

Reply:
Done. Can you take another look please?